gix-actor/src/signature/mod.rs

```diff
@@ -61,16 +63,10 @@ mod convert {
             Signature {
                 name: name.to_owned(),
                 email: email.to_owned(),
-                time: time.to_owned(),
+                time: Time::from_bytes(time).expect("Time must be valid"),
```

Comment:
This `From` implementation would also have to be removed now, but should probably be replaced with `TryFrom`.